test/integ/deadline_job_attachments/conftest.py

```diff
@@ -103,3 +104,13 @@ def external_bucket() -> str:
 
 def is_windows_non_admin():
     return sys.platform == "win32" and getpass.getuser() != "Administrator"
+
+
+def is_Windows_file_path_limit():
```

Comment:
duplicate code? can it use the definition above?

Reply:
The reason is I do not care if it's Admin or not for this particular function. Which is why I have the sys.platform check

Reply:
Don't understand here, the code above is 100% the same 

```
    if sys.platform == "win32":
        ntdll = ctypes.WinDLL("ntdll")
        ntdll.RtlAreLongPathsEnabled.restype = ctypes.c_ubyte
        ntdll.RtlAreLongPathsEnabled.argtypes = ()

        return bool(ntdll.RtlAreLongPathsEnabled())
    return True
```